K-means and Gaussian Mixture Model/k_means.py

```diff
@@ -71,6 +71,7 @@ def main():
         plt.scatter(df['x'], df['y'], c=labels1, cmap='viridis')
         plt.xlim(-4, 8)
         plt.ylim(-4, 8)
+        plt.savefig('./plots/kmeans.png')
```

Comment:
Thanks for the PR, but this line will throw an error: no such directory exists.
We Can change this line to `plt.savefig('kmeans.png')`

Reply:
Also, you could add this to line number 62. Because this plot is for Gaussian Mixture Model. Here you can add `plt.savefig('gaussianMixtureModel.png')`